src/components/PeerGroupsUpdate.tsx

```diff
@@ -0,0 +1,193 @@
+import React, {useEffect, useRef, useState} from 'react';
+import {useDispatch, useSelector} from "react-redux";
+import {RootState} from "typesafe-actions";
+import { actions as peerActions } from '../store/peer';
```

Comment:
Codacy has a fix for the issue: [Replace `'../store/peer'` with `"../store/peer"`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
import { actions as peerActions } from "../store/peer";
```